applications/nrf5340_audio/src/modules/audio_sync_timer_rtc.c

```diff
@@ -160,6 +161,29 @@ static int audio_sync_timer_init(void)
 		return ret;
 	}
 
+	/* Initialize capturing of current timestamps */
+	ret = nrfx_dppi_channel_alloc(&dppi_channel_curr_time_capture);
+	if (ret - NRFX_ERROR_BASE_NUM) {
+		LOG_ERR("nrfx DPPI channel alloc error (I2S frame start) - Return value: %d", ret);
+		return ret;
```

Comment:
Suggest we translate the error here, and use standard Zephyr err codes.

Reply:
done